codegen/winterfell/src/air/boundary_constraints.rs

```diff
@@ -45,18 +47,17 @@ pub(super) fn add_fn_get_aux_assertions(impl_ref: &mut Impl, ir: &Air) {
 /// trace segment
 fn add_main_trace_assertions(func_body: &mut codegen::Function, ir: &Air) {
     let elem_type = ElemType::Base;
```

Comment:
Do we actually need to define this variable? Seems like we only use it once in the code below and we could easily replace that with `ElemType::Base`.